mava/utils/make_env.py

```diff
@@ -61,7 +66,10 @@ def add_optional_wrappers(
 ) -> Environment:
     # Add the global state to observation.
     if add_global_state:
-        env = GlobalStateWrapper(env)
+        if hasattr(env, "has_global_state"):
+            env.has_global_state = True
+        else:
+            env = GlobalStateWrapper(env)
```

Comment:
I assume this is will change once we merge the global state fix PR?

Reply:
Yeah this is just temporary, we should merge the fix first!